shared/team-building/index.tsx

```diff
@@ -238,9 +239,12 @@ class TeamBuilding extends React.PureComponent<Props, {}> {
     sections: Array<SearchRecSection>,
     indexInList: number
   ): {index: number; length: number; offset: number} => {
+    const sectionLength = Kb.SectionDivider.height
+    const rowLength = userResultHeight
```

Comment:
Let's use `height` instead of length. I'm realizing it'd be a lot easier to understand the code with that simple change.

Reply:
sounds good. react-native return type here is `{ index, offset, length }` but I agree that "length" can be misleading in lists growing vertically